shop/src/main/java/ro/msg/learning/shop/Controller/OrdersController.java

```diff
@@ -0,0 +1,28 @@
+package ro.msg.learning.shop.Controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ro.msg.learning.shop.DTO.CreateOrderDto;
+import ro.msg.learning.shop.Domain.Orders;
+import ro.msg.learning.shop.Service.Implementation.OrderService;
+
+@RequestMapping("/order")
+@RestController
+public class OrdersController {
+    @Autowired
+    private OrderService orderService;
+
+    @PostMapping
+    public ResponseEntity<Orders> createOrder(@RequestBody CreateOrderDto createOrderDto) {
+        try {
```

Comment:
We could avoid adding catch blocks for every method by using a controller advice: https://www.baeldung.com/exception-handling-for-rest-with-spring#controlleradvice